pwa/src/serviceWorker.ts

```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore file */
```

Comment:
I gather that this is for test coverage?

Reply:
Correct, meaning it's for the test coverage to be accurate. We don't utilize custom logic in the service worker, so there's no need to test.